src/pages/index.js

```diff
@@ -1,12 +1,27 @@
 import * as React from "react";
 import { DefaultLayout } from "../components/DefaultLayout";
+import { Button } from "../components/Button";
+import { Container } from "../components/Container";
 
 const IndexPage = () => {
   return (
-    <main className="container m-auto">
+    <Container>
       <title>Portfolio page</title>
-      <DefaultLayout></DefaultLayout>
-    </main>
+      <DefaultLayout>
+        <div className="grid grid-cols-1 lg:grid-cols-2 gap-x-32 my-56">
+          <div>left content</div>
+          <div className="grid">
```

Comment:
This element doesn't have to be a `grid`. You can remove this class.

Reply:
but if I delete the "grid" property then "grid-cols-2" not going to work either.
what do you think?

Reply:
It will still work. `grid grid-cols-2` on the _parent_ will affect the children, but each child doesn't have to have the `grid` class.

Reply:
Sorry. I did not see the 13 line 
```<div className="grid">```
I thought you spoke about the 11 one
``` <div className="grid grid-cols-1 lg:grid-cols-2 gap-x-32 my-56">```